src/python/pants/backend/awslambda/python/rules.py

```diff
@@ -86,10 +87,15 @@ async def package_python_awslambda(
         ],
     )
 
+    lockfile_digest = None
+    if lambdex.lockfile != "<none>":
+        lockfile_request = await Get(PythonLockfileRequest, LambdexLockfileSentinel())
+        lockfile_digest = lockfile_request.hex_digest
```

Comment:
Maybe `lockfile_hash`? Now that I see this in an `@rule`, the word "digest" is overloaded to think "directory digest" from `pants.engine.fs`.

I think this could get really confusing, e.g. people accidentally using it with `MergeDigests`

Reply:
I still think that throwing around the word `hash` in a Python database is likely to bring up similar confusion. That said, [these lines from our JVM support](https://github.com/pantsbuild/pants/blob/main/src/python/pants/jvm/resolve/coursier_fetch.py#L391-L395) make `digest` on its own a non-starter for our lockfile support.

wheeeeeeeeeeeee

Reply:
Oh yeah true. Then, maybe being explicit is the key. `invalidation_digest` or `invalidation_hash`, but never just `hash` or just `digest`.

Reply:
I'll go with `hex_digest`, since that's what I'm using elsewhere.